src/Timers.chpl

```diff
@@ -0,0 +1,37 @@
+use Time, AllLocalesBarriers;
+
+pragma "default intent is ref"
+record enumTimers {
+  type enumT;
+  param enabled = true;
```

Comment:
Will all of those code be folded out when this param is false? (i.e., will this affect the program in any way when enabled is set to false?)

Reply:
Correct, it will all be folded out when `enabled=false` and should not impact the program in any way.